### PR TITLE
Add support for PIV card logins and disable debug logging

### PR DESCRIPTION
**Pull request by kryswisnaskas**
## Description of change
Adds support for the data format returned from the user-info endpoint from HSES for the PIV card user logins. After the HSES upgrade, while the username/password user logins worked, the PIV card users were not since the format of the data changed. This PR also points to the new HSES url for staging/dev/sandbox environments and disables the extra logging.

## How to test
On sandbox:
- verify that login into the hub works and the app behaves as expected

Or locally:
- update the .`env `file with the new HSES url (`AUTH_BASE=https://staging.hses.ohs.acf.hhs.gov`)
- set `BYPASS_AUTH=false`
- set the `AUTH_CLIENT_ID` and `AUTH_CLIENT_SECRET`  to new values (I'll send via keybase)
- verify that login into the hub works and the app behaves as expected

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-176

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Comment by thewatermethod** [comment]
@kryswisnaskas if you want to send me the keys on keybase to test, my keybase username is thewatermethod

File: .circleci/config.yml [hunk 1]
```diff
@@ -164,7 +164,7 @@ parameters:
     default: "main"
     type: string
   sandbox_git_branch:  # change to feature branch to test deployment
-    default: "add-wysisyg-editor"
+    default: "kw-fix-piv-card-login"
```
Comment (AdamAdHocTeam) [inline .circleci/config.yml:167]: Not sure why this was changed. Did we loose what ever purpose the previous value served?

File: deployment_config/dev_vars.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ web_memory: 512M
 worker_instances: 1
 worker_memory: 512M
 LOG_LEVEL: info
-AUTH_BASE: https://uat.hsesinfo.org
+AUTH_BASE: https://staging.hses.ohs.acf.hhs.gov
```
Comment (AdamAdHocTeam) [inline deployment_config/dev_vars.yml:7]: Again I'm assuming this change from UAT > Staging was intentional.

File: src/app.js [hunk 3]
```diff
@@ -59,15 +59,23 @@ app.get(oauth2CallbackPath, async (req, res) => {
...
-    const {
-      name,
-      principal: {
-        username,
-        userId,
-        authorities,
-      },
-    } = data;
+
+    logger.debug(`User details response data: ${JSON.stringify(data, null, 2)}`);
```
Comment (AdamAdHocTeam) [inline src/app.js:63]: Changed from audit logger to debug logger I'm assuming this was intentional.

**Review by AdamAdHocTeam — APPROVED** [review]
Approved just give a second look at some of the items commented below.

**Pull request merged**